### ISSUE TITLE
compliment and question

### ISSUE THREAD

**Issue by Schecher1** [wine-gameservers/arch-wine-fs22#1]
I wanted to thank you first of all for this nice docker image,  at first I thought it was just lovelessly slapped there, but the documentation **is very good** & later the VNC UI **surprised me** so much.
I had zero problems.

One question, why did **you use Arch**? 

It is in my interest for another game to also make such a server, this would be Space Engineers because it is also only available as a WIN option & there are so far only options where the ingame **game is very laggy** _(possibly the next project for you)._

**In summary:** Did you use **Arch because Wine behaves bette**r there or just because you are **an Arch guy**?

Greetings

**Comment by Toetje585** [comment]
Hello, 

I was inspired by images from https://github.com/binhex, the logic the scripts the way he uses init. Also arch is a rolling release so the packages are mostly always the latest available. I also switched to ArchLinux as daily driver because they have the best documentation around off all the distros. And I love to build my own packages, e.g. mesa/kernel for my hardware instead of the default generic profiles.

However images in debian/ubuntu will be as good in terms of performance.  I don't think a server in linux for Space Engineers would have better performance then on Windows, however it will be sure a bit lighter in terms of the the resources you need.

Best Regards,

**Comment by Schecher1** [comment]
Thanks for the quick reply. 
Oh, good to know, I am a Debian lover.

Regarding "I don't think a server in linux for Space Engineers would have better performance then on Windows", no, I didn't want to get better performance but either as good as on Windows or a bit worse.

For example, "https://github.com/mmmaxwwwell/space-engineers-dedicated-docker-linux" works and starts fine but... 
But it runs maximum bad, 2 players are enough to get noticeable lags, so that playing becomes impossible (or not enjoyable).
